### PR TITLE
fix: l10n string imports

### PR DESCRIPTION
**Pull request by firebomb1**
## Description

I've noticed that we have not been loading all the languages for asset description overrides for some time. Fixing this and fixing the order of imports to match both alphabetical order for translations and English as the first import (as they the default strings, they ought to be requested/imported first).

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Risk
Low, same imports with few additional languages for the asset descriptions.

## Testing

Check that the asset descriptions for Ethereum and FOXy (for example) are now translated in all languages (except Russian, which has not been translated yet).

### Engineering
☝️ 

### Operations
☝️ 

## Screenshots (if applicable)
Before:
image

After:
image

### CODE REVIEW

File: src/lib/asset-service/service/descriptions/index.ts [hunk 1]
```diff
@@ -1,12 +1,16 @@
+// eslint-disable-next-line simple-import-sort/imports -- keep English import on top
 import en from './en.json'
+import de from './de.json'
 import es from './es.json'
 import fr from './fr.json'
-import id from './id.json'
-import ko from './ko.json'
+// import id from './id.json'
+// import ko from './ko.json'
```
Comment (woodenfurniture) [inline src/lib/asset-service/service/descriptions/index.ts:7]: we can re-import these later if need be
```suggestion
```

File: src/assets/translations/index.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // eslint-disable-next-line simple-import-sort/imports -- keep English import on top
 import en from './en/main.json'
-import es from './es/main.json'
 import de from './de/main.json'
+import es from './es/main.json'
 import fr from './fr/main.json'
 // import id from './id/main.json'
 // import ko from './ko/main.json'
```
Comment (woodenfurniture) [inline src/assets/translations/index.ts:7]:
```suggestion
```

**Review by woodenfurniture — APPROVED** [review]
LY for the PR ser

**Review by gomesalexandre — APPROVED** [review]
LGTM!

**Pull request merged**